twoopstracker/settings.py

```diff
@@ -229,7 +229,15 @@
     "TWOOPSTRACKER_CONFIRM_EMAIL_URL", "http://localhost:3000/api/auth/confirm-email"
 )
 
+# Custom Register Serializer
+# https://dj-rest-auth.readthedocs.io/en/latest/configuration.html
+REST_AUTH_REGISTER_SERIALIZERS = {
+    "REGISTER_SERIALIZER": "twoopstracker.authentication.serializers.CustomRegisterSerializer",
+}
 
+DEFAULT_FROM_EMAIL = env.str(
+    "DEFAULT_FROM_EMAIL", "no-reply@trolltracker.investigate.africa"
```

Comment:
```suggestion
    "DEFAULT_FROM_EMAIL", "webmaster@localhost"
```

... after all, the project is `TwoopsTracker`, where does `TrollTracker` even come from 😎